aspnetcore/security/enforcing-ssl.md

```diff
@@ -1,37 +1,39 @@
 ---
-title: Enforcing SSL in an ASP.NET Core app
+title: Enforcing TLS in an ASP.NET Core app
 author: rick-anderson
-description: Shows how to require SSL in a ASP.NET Core web app
+description: Shows how to require HTTPS/TSL in a ASP.NET Core web app
 manager: wpickett
 ms.author: riande
-ms.date: 07/19/2017
+ms.date: 2/9/2018
 ms.prod: asp.net-core
 ms.technology: aspnet
 ms.topic: article
 uid: security/enforcing-ssl
 ---
-# Enforcing SSL in an ASP.NET Core app
+# Enforcing HTTPS/TSL in an ASP.NET Core app
 
 By [Rick Anderson](https://twitter.com/RickAndMSFT)
 
 This document shows how to:
 
-- Require SSL for all requests (HTTPS requests only).
+- Require TLS for all requests (HTTPS requests only).
 - Redirect all HTTP requests to HTTPS.
 
-## Require SSL
+> [!WARNING]
+> Do not use [RequireHttpsAttribute](https://docs.microsoft.com/aspnet/core/api/microsoft.aspnetcore.mvc.requirehttpsattribute) on Web APIs that receive sensitive information. `RequireHttpsAttribute` does transparent redirection from HTTP to HTTPS so the client never knows the information was sent insecurely over HTTP. Web APIs should close the connection with status code 400 (Bad Request) and not serve the request.
 
-The [RequireHttpsAttribute](https://docs.microsoft.com/aspnet/core/api/microsoft.aspnetcore.mvc.requirehttpsattribute) is used to require SSL. You can decorate controllers or methods with this attribute or you can apply it globally as shown below:
+<a name="require-ssl"></a>
+## Require TLS
 
-Add the following code to `ConfigureServices` in `Startup`:
+The [RequireHttpsAttribute](https://docs.microsoft.com/aspnet/core/api/microsoft.aspnetcore.mvc.requirehttpsattribute) is used to require TSL. You can decorate controllers or methods with this attribute or you can apply it globally. To apply the attribute globally, add the following code to `ConfigureServices` in `Startup`:
 
 [!code-csharp[Main](authentication/accconfirm/sample/WebApp1/Startup.cs?name=snippet2&highlight=4-999)]
 
-The highlighted code above requires all requests use `HTTPS`, therefore HTTP requests are ignored. The following highlighted code redirects all HTTP requests to HTTPS:
+The preceding highlighted code requires all requests use `HTTPS`, therefore HTTP requests are ignored. The following highlighted code redirects all HTTP requests to HTTPS:
```

Comment:
therefore :point_right: A semi-colon in front and comma after it. ... or ... Change comma to period and start a new sentence with a comma after "Therefore" in the new sentence.